docs/concepts/storage/volumes.md

```diff
@@ -563,6 +564,9 @@ PersistentVolume `nodeAffinity` is required when using local volumes. It enables
 the Kubernetes scheduler to correctly schedule pods using local volumes to the
 correct node.
 
+PersistentVolume `volumeMode` can now be set to "Block" (instead of the default
```

Comment:
Mention that volumeMode needs BlockVolume alpha feature enabled.

Reply:
Fixed.